src/Dropdown/Dropdown.test.jsx

```diff
@@ -168,6 +174,8 @@ describe('<Dropdown />', () => {
 
       it('does not toggle with invalid key', () => {
         wrapper = mount(<Dropdown {...props} />);
+        // resetting focus
+        wrapper.find('Button').getDOMNode().focus();
```

Comment:
why we have to make this change in existing test?

Reply:
this test started failing on [line](https://github.com/edx/paragon/blob/master/src/Dropdown/Dropdown.test.jsx#L180)  after updating code in this PR.
**Question**: Why it was running before?
**Answer**: Before starting the test `does not toggle with invalid key` `'Button'` lost focus.
Our previous DropDown code was focusing Button on every update and `Button` was getting focus automatically before [line](https://github.com/edx/paragon/blob/master/src/Dropdown/Dropdown.test.jsx#L180) .
As now Button Don't get focus automatically we had to reset focus.